### PR TITLE
Add io.github.gerryferdinandus.bittorrent-tracker-editor

### PR DESCRIPTION
**Pull request by GerryFerdinandus**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

This editor should be able to write/read torrent files that can be placed anywhere in 'home'
`- --filesystem=home`

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build io.github.gerryferdinandus.bittorrent-tracker-editor

File: io.github.gerryferdinandus.bittorrent-tracker-editor.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  - --socket=x11
+  - --env=QT_QPA_PLATFORM=xcb
+  # Allow loading/saving files from anywhere
+  - --filesystem=home
+  # Download bittorrent trackers
+  - --share=network
+  # Standard permissions
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
```
Comment (hfiguiere) [inline io.github.gerryferdinandus.bittorrent-tracker-editor.yml:19]: why does it need access to sound?

**Comment by GerryFerdinandus** [comment]
bot, build io.github.gerryferdinandus.bittorrent-tracker-editor

**Comment by GerryFerdinandus** [comment]
both test builds were tested successfully
Raspberry pi 4 (arm64): OK
PC (amd64): OK

File: io.github.gerryferdinandus.bittorrent-tracker-editor.yml (hunk 1 above)
Comment (GerryFerdinandus) [reply io.github.gerryferdinandus.bittorrent-tracker-editor.yml]: is now removed

**Comment by barthalion** [comment]
bot, build io.github.gerryferdinandus.bittorrent-tracker-editor

**Pull request closed without merging**